frontend/src/assets/styles/_common.scss

```diff
@@ -5,3 +5,15 @@
 .vert-align-middle {
     vertical-align: middle;
 }
+
+.pos-rel {
```

Comment:
I probably should've said this is my last PR review. Usually what I've seen is grouping CSS classes by components, not by individual styles. Kinda like this: 
```/****** header ******/
styles goes here...
/****** navigation ******/
styles goes here...
/****** main content ******/
styles goes here...
/****** footer ******/
styles go here...```

I grabbed that from number 3 of this article:
https://www.tothenew.com/blog/10-best-practices-in-css/